src/npcmove.cpp

```diff
@@ -428,6 +445,10 @@ void npc::assess_danger()
     int hostile_count = 0;
     int friendly_count = 1; // count yourself as a friendly
     int def_radius = rules.has_flag( ally_rule::follow_close ) ? follow_distance() : 6;
+    float NPC_COWARDICE_MODIFIER = get_option<float>( NPC_COWARDICE_MODIFIER_OPT );
+    float NPC_MONSTER_DANGER_MAX = get_option<float>( NPC_MONSTER_DANGER_MAX_OPT );
+    float NPC_CROWD_BRAVADO = get_option<int>( NPC_CROWD_BRAVADO_OPT );
+    float NPC_DANGER_VERY_LOW = get_option<float>( NPC_DANGER_VERY_LOW_OPT );
```

Comment:
Looks like this function could be called very often so it might make sense to cache these option values.

Reply:
Yes, I agree. I don't know how to do that unfortunately.

Reply:
You can add the options to `cached_options.h/cpp` and cache the options in the following function:

https://github.com/CleverRaven/Cataclysm-DDA/blob/f5fe4d99f38193623fcc202ee1e3051760ba29fa/src/options.cpp#L3995-L4027

Reply:
Thanks, I should be able to cargo-cult it from there when I get home.